package/MDAnalysis/converters/RDKit.py

```diff
@@ -53,7 +53,7 @@
     infers the structures of approximately 90% of the `ChEMBL27`_ dataset.
     Work is currently ongoing on further improving this and updates to the
     converter are expected in future releases of MDAnalysis.
-    Please see `Pull Request #3044`_ for more details.
+    Please see `Pull Request #3044`_ and `Issue #3339`_ for more details.
```

Comment:
I think the link to the PR can be removed. Better write some text that describes the current capabilities and lists the remaining known limitations. If there's an issue for these limitations, link to the issue.